### ISSUE TITLE
Minimum Stability issue

### ISSUE THREAD

**Issue by dwhoban** [nystudio107/craft3-imageoptimize#5]
Getting the following error when installing on clean craft3.beta 12 setup

``` zsh
$ composer require nystudio107/craft3-ImageOptimize -vvv
Reading ./composer.json
Loading config file ./composer.json
Checked CA file /private/etc/ssl/cert.pem: valid
Executing command (/Users/dan/Sites/hoban): git branch --no-color --no-abbrev -v
Executing command (/Users/dan/Sites/hoban): git describe --exact-match --tags
Executing command (/Users/dan/Sites/hoban): git log --pretty="%H" -n1 HEAD
Executing command (/Users/dan/Sites/hoban): hg branch
Executing command (/Users/dan/Sites/hoban): fossil branch list
Executing command (/Users/dan/Sites/hoban): fossil tag list
Executing command (/Users/dan/Sites/hoban): svn info --xml
Reading /Users/dan/.composer/composer.json
Loading config file /Users/dan/.composer/composer.json
Reading /Users/dan/Sites/hoban/vendor/composer/installed.json
Loading plugin yii\composer\Plugin
Loading plugin craft\composer\Plugin
Running 1.4.1 (2017-03-10 09:29:45) with PHP 7.0.17 on Darwin / 16.5.0
Downloading https://asset-packagist.org/packages.json
Writing /Users/dan/.composer/cache/repo/https---asset-packagist.org/packages.json into cache
Downloading https://packagist.org/packages.json
Writing /Users/dan/.composer/cache/repo/https---packagist.org/packages.json into cache
Reading /Users/dan/.composer/cache/repo/https---asset-packagist.org/p-provider-latest-.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2013.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2014.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2015.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2016.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2016-07.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2016-10.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2017-01.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-2017-04.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-archived.json from cache
Reading /Users/dan/.composer/cache/repo/https---packagist.org/p-provider-latest.json from cache


  [InvalidArgumentException]
  Could not find package nystudio107/craft3-ImageOptimize at any version for your minimum-stability (beta). Check the package spelling or your minimum-stability


Exception trace:
 () at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/src/Composer/Command/InitCommand.php:643
 Composer\Command\InitCommand->findBestVersionForPackage() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/src/Composer/Command/InitCommand.php:365
 Composer\Command\InitCommand->determineRequirements() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/src/Composer/Command/RequireCommand.php:117
 Composer\Command\RequireCommand->execute() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/vendor/symfony/console/Command/Command.php:267
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/vendor/symfony/console/Application.php:846
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/vendor/symfony/console/Application.php:191
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/src/Composer/Console/Application.php:227
 Composer\Console\Application->doRun() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/vendor/symfony/console/Application.php:122
 Symfony\Component\Console\Application->run() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/src/Composer/Console/Application.php:100
 Composer\Console\Application->run() at phar:///usr/local/Cellar/composer/1.4.1/libexec/composer.phar/bin/composer:54
 require() at /usr/local/Cellar/composer/1.4.1/libexec/composer.phar:24

require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-suggest] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--] [<packages>]...
```
Other NYStudio107 plugins installing, just not this one.

Have tried shifting minimum stability down to dev with the same error.

**Comment by khalwat** [comment]
I'm reasonably sure that the `composer.json` is correct; are you able to use _any_ of the plugins up here?

https://github.com/craftcms/plugins

Can you post your Craft project `composer.json`?

**Comment by khalwat** [comment]
Ahhhh... nevermind. Someone had me rename the plugin, and I didn't do it properly. I will fix.

**Comment by khalwat** [comment]
Fixed in 1.0.8: https://github.com/nystudio107/craft3-imageoptimize/releases/tag/1.0.8
